### DISCUSSION TITLE
Aggregation function sum error

### DISCUSSION THREAD

**Discussion by chirdeeptomar** [finos/TimeBase-CE#211] [Q&A]
Hello,

We are trying to sum up the volume and getting a 500 on the Web Admin UI without any notification. 

```
WITH entries[THIS IS com.epam.deltix.qsrv.hf.tickdb.ui.tbshell.SimpleBarMessage] AS 'entries'
SELECT sum{}(sum(entries.volume)) AS 'volume'
FROM bars
WHERE timestamp BETWEEN '2005-03-01 14:00:00.000'd AND '2005-03-01 15:00:00.000'd and symbol=='AAPL'

```
Error in shell

```
==> WITH entries[THIS IS com.epam.deltix.qsrv.hf.tickdb.ui.tbshell.SimpleBarMessage] AS 'entries'
SELECT sum{}(sum(entries.volume)) AS 'volume'
FROM bars
WHERE timestamp BETWEEN '2005-03-01 14:00:00.000'd AND '2005-03-01 15:00:00.000'd and symbol=='AAPL'
>>> Error: WITH: unrecognized command. (Type ? for usage)
==> >>> Error: CompilationException: 1.18..25: UnknownIdentifierException: Unknown identifier: ENTRIES
==> >>> Error: FROM: unrecognized command. (Type ? for usage)
==> >>> Error: WHERE: unrecognized command. (Type ? for usage)
```

Can you please advise, what we might be doing wrong?

Thanks

**Comment by alex-karpovich** [comment]
Сould you please share stream schema? I think such types of queries applicable for our 'Universal' data format only: 
For messages like this [PackageHeader.java.txt](https://github.com/finos/TimeBase-CE/files/10683260/PackageHeader.java.txt)

**Reply by chirdeeptomar** [reply]
Its the sample stream "bars" given by TimeBase. 

```
set db dxtick://localhost:8011 
open
generate bars 
exit
```

**Reply by chirdeeptomar** [reply]
We were able to figure out the error. Admin UI is still broken as doesn't give out any errors and you have to look at the developer console in the browser.

**Reply by alex-karpovich** [reply]
Please try this project https://github.com/epam/TimebaseCryptoConnectors

to be able to receive data from crypto exchanges in L2 data format.

**Reply by alex-karpovich** [reply]
> Its the sample stream "bars" given by TimeBase.
> 
> ```
> set db dxtick://localhost:8011 
> open
> generate bars 
> exit
> ```

for the 'bars' stream you can do only simple selects like aggregation, count, etc. 
bars contains only OHLC fields, without complex structures like arrays/objects compare to L2 data
